encoding/v2/kzg/verifier/verifier.go

```diff
@@ -277,9 +277,8 @@ func genRhsG1(
 // Each sample need not have unique row, it is possible that multiple chunks of the same blob are validated altogether
 func (v *Verifier) universalVerify(params encoding.EncodingParams, samples []Sample, numBlobs int) error {
 	// precheck
-	for i, s := range samples {
+	for _, s := range samples {
 		if s.RowIndex >= numBlobs {
-			fmt.Printf("sample %v has %v Row, but there are only %v blobs\n", i, s.RowIndex, numBlobs)
 			return errors.New("sample.RowIndex and numBlob are inconsistent")
```

Comment:
prob want to incorporate the print in the error? otherwise error will not have enough info right?